### ISSUE TITLE
Support multiple open windows

### ISSUE THREAD

**Issue by MPLew-is** [GitFiend/Support#124]
Is it possible to add support for multiple windows/instances of GitFiend? I like to have a desktop for each related group of repos I work on, so being able to open multiple windows would be great.

**Comment by GitFiend** [comment]
Yes, this is something I want to add. I disabled it early in development to keep things simple, but I may take a look at it again soon.
